snet_cli/utils.py

```diff
@@ -183,3 +183,13 @@ def compile_proto(entry_path, codegen_dir, proto_file=None):
 
     except Exception as e:
         return False
+
+# return element of abi (return None if fails to find)
+def abi_get_element_by_name(abi, name):
+    for a in abi["abi"]:
```

Comment:
Will "abi" key always be within abi dict?
If not or abi == None, this for loop will break.
I'd suggest use something like `if abi and "abi" in abi:`
Offtopic: There are too much off `abi` in this line...

Reply:
done :)